tests/src/Kernel/fixtures/rendering.yml

```diff
@@ -3243,3 +3243,125 @@
     equals:
       'div.ecl-list-illustration__item:nth-child(1) svg.ecl-icon': '<use xlink:href="/themes/custom/oe_theme/dist/ec/images/icons/sprites/icons.svg#growth"></use>'
       'div.ecl-list-illustration__item:nth-child(2) svg.ecl-icon': '<use xlink:href="/themes/custom/oe_theme/dist/ec/images/icons/sprites/icons.svg#budget"></use>'
+- array:
```

Comment:
Why cannot use single tests with all these items?
Also looks like we didn't test the 'counter_label' option in the pattern.

Reply:
I'll merge 2 of the test array but we still need to keep 2 for testing the default value for the `counter_label` and `full_width`